config.md

```diff
@@ -59,6 +59,7 @@ For Solaris, the mount entry corresponds to the 'fs' resource in the [zonecfg(1M
 
 * **`destination`** (string, REQUIRED) Destination of mount point: path inside container.
   This value MUST be an absolute path.
+  * Posix: the path and child directories MUST exist, a runtime MUST NOT create directories automatically to a mount point.
```

Comment:
> Right, but currently it is incorrect, as it states this in config.go for Windows where that statement is incorrect.

The [Markdown is normative, and the Go comments are not][1].  I'm in favor of removing the incorrect Go comment (like you do in this PR), and in general making the Go comments as simple as possible to avoid this sort of problem.  But I don't want to carry incorrect Go comments back into the normative Markdown (like you do in this hunk).

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc5/config.md#L4